src/Microsoft.AspNetCore.Razor.Sdk/SDK/Sdk.props

```diff
@@ -0,0 +1,17 @@
+<!--
+***********************************************************************************************
+Sdk.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="$(MSBuildThisFileDirectory)..\build\netstandard2.0\Sdk.CurrentVersion.props"
```

Comment:
Can we give this a filename that has Razor in it? `Sdk.CurrentVersion.props` vs Razor.Sdk.CurrentVersion.props`